File: BEE 1038.py
code, items = map(int, input().split())

if code == 1:
    print("Total: R$ {:.2f}".format(4.0 * items))
elif code == 2:
    print("Total: R$ {:.2f}".format(4.5 * items))
elif code == 3:
    print("Total: R$ {:.2f}".format(5.0 * items))
elif code == 4:
    print("Total: R$ {:.2f}".format(2.0 * items))
elif code == 5:
    print("Total: R$ {:.2f}".format(1.5 * items))

File: BEE 1038.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');
var sep = [' ', '\n'];
// console.log(separators.join('|'));
var line = input.split(new RegExp(sep.join('|'), 'g'));

var code = parseInt(line.shift());
var items = parseInt(line.shift());

if (code == 1){
    console.log("Total: R$", (4.0 * items).toFixed(2));
}
else if (code == 2){
    console.log("Total: R$", (4.50 * items).toFixed(2));
}
else if (code == 3){
    console.log("Total: R$", (5.0 * items).toFixed(2));
}
else if (code == 4){
    console.log("Total: R$", (2.0 * items).toFixed(2));
}
else if (code == 5){
    console.log("Total: R$", (1.5 * items).toFixed(2));
}